style.css

```diff
@@ -82,7 +85,14 @@ a {
   }
 
   .nav-list .close-button {
-    color: #000;
+    background-color: transparent;
+    border: none;
+    color: #fff;
+    cursor: pointer;
+    font-size: 18px;
+    padding: 0;
+    margin-top: 30px;
+    margin-right: 100px;
   }
 
   .nav-list li {
```

Comment:
- [ ] Nice work implementing the mobile menu :clap: However it would be nice to reduce the margin between the mobile menu options and add more padding at left of the mobile menu to follow the Figma design.
### Your design :point_down: 
![Screenshot from 2023-05-23 09-28-18](https://github.com/Eccb7/Portfolio2.0/assets/91142432/4be9da60-19bc-43c4-985c-1b9f22fe843e)
### What is supposed to look like :point_down: 
![Screenshot from 2023-05-15 17-26-08](https://github.com/Eccb7/Portfolio2.0/assets/91142432/553890f2-491b-402f-92be-7ccdb9925c2e)